packages/@atjson/source-html/src/annotations/anchor.ts

```diff
@@ -0,0 +1,11 @@
+import { InlineAnnotation } from '../../../document/dist/commonjs';
```

Comment:
Isn't `lerna bootstrap` meant to free us from relatively importing a colocated package?

Reply:
Urgh. This is a mistake